internal/elasticsearch/nodestats.go

```diff
@@ -0,0 +1,153 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package elasticsearch
+
+import (
+	"encoding/json"
+	"io"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/elasticsearch/ingest"
+)
+
+// StatsRecord contains stats for a measurable entity (pipeline, processor, etc.)
```

Comment:
Thanks for moving it to the `elasticsearch` package. I'm looking at `nodestats` implementation and I'm wondering if `nodestats` exposes in the future anything else than processors or pipelines, so mainly ingest. Maybe we should put it in the `elasticsearch/ingest`? WDYT?